packages/@ourworldindata/grapher/src/mapCharts/ChoroplethGlobe.tsx

```diff
@@ -0,0 +1,622 @@
+        // where possible. See https://www.jasondavies.com/maps/rotate/ for more
+        // details.
+        //
+        // We could rely on D3's event.transform.k for zooming, but the
+        // transform value might be out of sync with the actual zoom level if
+        // the zoom level was changed elsewhere (e.g. by automatically zooming
+        // in on a country). That's why we compute the target zoom level ourselves.
+
+        const panAndZoom = (): any => {
+            let previousType: InteractionType | undefined
+
+            // for panning
+            let startCoords: [number, number, number],
+                startQuat: [number, number, number, number],
+                startRot: [number, number, number]
+
+            // for zooming
+            let startDistance: number | undefined
+
+            const getInteractionType = (event: any): InteractionType => {
+                if (event.sourceEvent.type === "wheel") return "zoom-scroll"
+                if (isMultiTouchEvent(event.sourceEvent)) return "zoom-pinch"
+                return "pan"
+            }
+
+            const panningOrZoomingStart = (event: any): void => {
+                const type = getInteractionType(event)
+
+                const startPinching = (): void => {
+                    startDistance = calculatePinchDistance(event.sourceEvent)
+                }
+
+                const startPanning = (): void => {
+                    const pos = getRelativeMouse(base, event.sourceEvent)
+                    startCoords = versor.cartesian(
+                        this.projection.invert([pos.x, pos.y])
+                    )
+                    startRot = this.projection.rotate()
+                    startQuat = versor(startRot)
+                }
+
+                if (type === "zoom-pinch") startPinching()
+                else if (type === "pan") startPanning()
+
+                previousType = type
+            }
+
+            const panningOrZooming = (event: any): void => {
+                this.isPanningOrZooming = true
+                this.clearHover() // dismiss the tooltip
+
+                const wheeling = (): void => {
+                    this.zoomGlobe(-event.sourceEvent.deltaY)
+                }
+
+                const pinching = (): void => {
+                    const distance = calculatePinchDistance(event.sourceEvent)
+
+                    if (!startDistance) {
+                        startDistance = distance
+                        return
+                    }
+
+                    const delta = distance - startDistance
+
+                    // We sometimes get two events for the same pinch gesture,
+                    // with one of the events having a delta of 0. We simply
+                    // ignore the delta-0 events. This fixes a bug where the
+                    // rendered SVG country paths would interfere with the
+                    // pinch-to-zoom gesture.
+                    if (delta === 0) return
```

Comment:
Thinking about this once more:
While this line here can obviously stay in, because we don't need to do anything about deltas of 0, in my testing I've also seen a bunch of times where we get two pinch events that have non-zero deltas (but where the second one is miniscule).
Which makes me think that we should make `zoomGlobe` more robust, which would mean that it can properly handle two incoming zoom events before the `requestAnimationFrame` has finished (by somehow storing the "target zoom level").
I think this will be beneficial, and will make it less likely that we behave weirdly in other browsers, or on low-perf devices, etc.

But only if that's relatively easy to do.